### DISCUSSION TITLE
Concatenate similar suggestions?

### DISCUSSION THREAD

**Discussion by pawamoy** [community/community#13239] [Other Feature Feedback, Questions, & Ideas]
Otherwise it's taking a lot of vertical space, moreover for things that don't really interest me (crosses).

![image](https://user-images.githubusercontent.com/3999221/159461244-a2025b88-956e-4c8a-8993-0ca9bcd21b55.png)

Thanks for allowing feedback, nice to see new designs 🙂

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
